docs/source/features/mixed_precision.rst

```diff
@@ -3,11 +3,21 @@
 Mixed Precision Training
 ------------------------
 
-Mixed precision training significantly enhances computational efficiency by conducting operations in half-precision and fp8 formats, while selectively maintaining minimal data in single-precision to preserve critical information throughout key areas of the network. NeMo now supports FP16, BF16, and FP8 (via Transformer Engine) across most models. Further details will be provided shortly.
+Mixed precision training significantly enhances computational efficiency by conducting operations in low-precision format, while selectively maintaining minimal data in single-precision to preserve critical information throughout key areas of the network. NeMo now supports FP16, BF16, and FP8 (via Transformer Engine) across most models. Further details will be provided shortly.
```

Comment:
Mixed precision training significantly improves computational efficiency by performing operations in low-precision formats. However, critical information is selectively maintained in single-precision to ensure accuracy in key areas of the network. NeMo now supports three precision modes: FP16, BF16, and FP8 (via the Transformer Engine) across most models. Additional details will be provided shortly.